modules/mobile/app/models/mobile/v0/adapters/military_information.rb

```diff
@@ -20,6 +20,7 @@ def parse(user_uuid, military_information)
 
         def format_service_period(service_period)
           if service_period[:branch_of_service].nil?
+            # no evidence of this happening. remove
             Rails.logger.warn(
               'mobile military information missing service period details', service_period:
             )
```

Comment:
Category: Data discovery
Action: remove. this isn't happening and it would probably be better to just let it fail now in order to surface the problem

Reply:
It seems I missed this in the first pass. Delete later.